backend/main.py

```
@@ -0,0 +1,121 @@
+            id=uuid4(),
+            source="sentry",
+            external_id=event_id,
+            title=title,
+            body=body,
+            metadata=payload,
+            created_at=datetime.now(timezone.utc)
+        )
+        add_feedback_item(item)
+        return {"status": "ok", "id": str(item.id)}
+
+    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)